packages/shared/package.json

```diff
@@ -93,6 +95,7 @@
     "@types/glob-to-regexp": "0.4.1",
     "@types/js-cookie": "3.0.2",
     "@types/node": "^18.17.0",
+    "cross-fetch": "^4.0.0",
```

Comment:
Is this only needed for the tests?

Reply:
Now that we're using >= Node 18 locally and on CICD as well, can we use something like this https://github.com/jsdom/jsdom/issues/1724#issuecomment-1446858041 ?

Reply:
Yep, only for tests 👍

I can shift to that JSDOM workaround if you feel it's preferable. The use of `cross-fetch` seems simpler than the custom patched environment, but it's fine if we want to minimize dependencies.

Reply:
No real preference here, both are good approaches